tests/lib/rules/multiline-comment-style.js

```diff
@@ -0,0 +1,379 @@
+                 */
+            `,
+            output: `
+                /*
+                 * the following line
+                 * is missing a '*' at the start
+                 */
+            `,
+            errors: [{ message: MISSING_STAR_ERROR, line: 4 }]
+        },
+        {
+            code: `
+                /*
+                 * the following line
+                      * has a '*' with the wrong offset at the start
+                 */
+            `,
+            output: `
+                /*
+                 * the following line
+                 * has a '*' with the wrong offset at the start
+                 */
+            `,
+            errors: [{ message: ALIGNMENT_ERROR, line: 4 }]
+        },
+        {
+            code: `
+                  /*
+                   * the following line
+                 * has a '*' with the wrong offset at the start
+                   */
+            `,
+            output: `
+                  /*
+                   * the following line
+                   * has a '*' with the wrong offset at the start
+                   */
+            `,
+            errors: [{ message: ALIGNMENT_ERROR, line: 4 }]
+        },
+        {
+            code: `
+                /*
+                 * the last line of this comment
+                 * is misaligned
+                   */
+            `,
+            output: `
+                /*
+                 * the last line of this comment
+                 * is misaligned
+                 */
+            `,
+            errors: [{ message: ALIGNMENT_ERROR, line: 5 }]
+        },
+        {
+            code: `
+                /*
+                 * foo
+                 * bar
+                 */
+            `,
+            output: `
+                // foo
+                // bar
+            `,
+            options: ["separate-lines"],
+            errors: [{ message: EXPECTED_LINES_ERROR, line: 2 }]
+        },
+        {
+            code: `
+                /* foo
+                 *bar
+                 baz
+                 qux*/
+            `,
+            output: `
+                // foo
+                // bar
+                // baz
+                // qux
+            `,
+            options: ["separate-lines"],
+            errors: [{ message: EXPECTED_LINES_ERROR, line: 2 }]
+        },
+        {
+            code: `
+                // foo
+                // bar
+            `,
+            output: `
+                /* foo
+                   bar */
+            `,
+            options: ["bare-block"],
+            errors: [{ message: EXPECTED_BLOCK_ERROR, line: 2 }]
+        },
+        {
+            code: `
+                /*
+                * foo
+                * bar
+                */
+            `,
+            output: `
+                /* foo
+                   bar */
+            `,
+            options: ["bare-block"],
+            errors: [{ message: EXPECTED_BLOCK_ERROR, line: 2 }]
+        }
```

Comment:
Can you add a test where a single-line comment contains `*/`? I think this will currently cause invalid syntax.

```js
/* eslint multiline-comment-style: [error, bare-block] */

// foo */
// bar

```